doc/classes/FontData.xml

```diff
@@ -179,6 +179,22 @@
 				Returns underline thickness in pixels.
 			</description>
 		</method>
+		<method name="get_variation" qualifiers="const">
+			<return type="float">
+			</return>
+			<argument index="0" name="tag" type="String">
+			</argument>
+			<description>
+				Returns variation coordinate [code]tag[/code].
+			</description>
+		</method>
+		<method name="get_variation_list" qualifiers="const">
+			<return type="Dictionary">
+			</return>
+			<description>
+				Returns [code]";"[/code] separated list of supported variation coordinates, each coordinate is returned in the following format: [code]"tag,min_value,max_value,default_value"[/code].
```

Comment:
Might be good to add some details on what variation coordinates are / which kind of fonts support them and for what purpose (e.g. with a link to an appropriate reference).

Reply:
Added description and link to OpenType registry.